spec/interfaceToC.dd

```diff
@@ -227,7 +227,7 @@ void foo(char[] string)
 ------
 
         $(P The $(CODE printf) format string literal
-        in the example doesn't end with $(CODE '\0').
+        in the example DOES end with $(CODE '\0').
```

Comment:
Full capitalization isn't used in spec. How about `$(I ...)` for italic?

Reply:
Changed to $(I does).